marimo/_cli/export/commands.py

```diff
@@ -101,6 +102,13 @@ async def start() -> None:
     type=bool,
     help="Include notebook code in the exported HTML file.",
 )
+@click.option(
+    "--reactive/--no-reactive",
+    default=False,
+    show_default=True,
+    type=bool,
+    help="Whether to export react HTML file, using marimo islands.",
```

Comment:
Is this supposed to say "reactive HTML file"? If so, "Whether to export a reactive HTML file; if enabled, the exported HTML will run Python"

We can omit `using marimo islands` from the description, since that is an implementation detail.

Reply:
Ah yes, good catch! I can push the change.